apps/browser/components/ChatLogs.tsx

```diff
@@ -61,56 +60,67 @@ export default function ChatLogs(props: ChatLogsProps) {
   }
 
   return (
-    <div
-      ref={listContainerRef}
-      onScroll={handleScroll}
-      className="w-full flex-1 items-center space-y-6 overflow-y-auto overflow-x-clip px-2 py-3 text-left"
-    >
-      {logs.map((msg, index, logs) => {
-        const isEvo = msg.user === "evo";
-        return (
-          <div key={index} className="m-auto w-full max-w-[56rem] self-center">
-            <div className="group relative flex w-full animate-slide-down items-start space-x-3 rounded-lg p-2 text-white opacity-0 transition-colors duration-300 ">
-              <div className="!h-8 !w-8 !min-w-[2rem]">
-                {isEvo && logs[index - 1].user === "user" ? (
-                  <Logo wordmark={false} className="w-full" chatAvatar />
-                ) : !isEvo ? (
-                  <>
-                    {session?.user.image && session?.user.email ? (
-                      <img
-                        src={session?.user.image}
-                        className="h-full w-full rounded-full bg-cyan-600"
-                      />
-                    ) : !session?.user.email ? (
-                      <Avatar size={32} />
-                    ) : (
-                      <div className="w-full rounded-full bg-cyan-600" />
-                    )}
-                  </>
-                ) : null}
-              </div>
-              <div className="max-w-[calc(100vw-84px)] space-y-2 pt-1 md:max-w-[49rem]">
-                {index === 0 || logs[index - 1].user !== msg.user ? (
-                  <div className="SenderName font-medium">
-                    {session?.user.name && !msg.user.includes("evo")
-                      ? session?.user.name
-                      : msg.user.charAt(0).toUpperCase() + msg.user.slice(1)}
-                  </div>
-                ) : null}
-                <div className="prose prose-invert w-full max-w-none">
-                  <ReactMarkdown>{msg.title.toString()}</ReactMarkdown>
-                  <ReactMarkdown>{msg.content?.toString() ?? ""}</ReactMarkdown>
+    <>
+      <div className="flex h-20 items-center justify-center border-b-2 border-zinc-800 md:h-12">
+        {/* <div>{chatName}</div> */}
```

Comment:
Can this be removed?

Reply:
This will be updated in this issue: https://github.com/polywrap/evo.ninja/issues/564